source/common/config/http_subscription_impl.h

```diff
@@ -75,24 +81,28 @@ class HttpSubscriptionImpl : public Http::RestApiFetcher,
     const auto status =
         google::protobuf::util::JsonStringToMessage(response.bodyAsString(), &message);
     if (status != google::protobuf::util::Status::OK) {
-      // TODO(htuch): Track stats and log failures.
+      ENVOY_LOG(warn, "REST config JSON conversion error: {}", status.ToString());
+      stats_.update_failure_.inc();
```

Comment:
Same comment here about local failure helper to keep the stat, etc. logic in 1 place.